packages/es-components/babel.config.js

```diff
@@ -12,26 +13,20 @@ module.exports = function(api) {
     ]
   ];
 
+  const removePropTypesPlugin = [
+    'babel-plugin-transform-react-remove-prop-types',
+    { removeImport: true }
+  ];
   const plugins = [
     ['@babel/plugin-transform-runtime', { corejs: 3 }],
     '@babel/plugin-proposal-export-default-from',
     '@babel/plugin-proposal-optional-chaining'
-  ];
+  ].concat(inProd ? [removePropTypesPlugin] : []);
```

Comment:
This is basically the same as using env, but the plugin merging is more explicit.